src/main/scala/org/bblfsh/client/BblfshClient.scala

```diff
@@ -52,6 +52,10 @@ class BblfshClient(host: String, port: Int, maxMsgSize: Int) {
     stub.version(req)
   }
 
+  def shutdownNow(): Unit = {
```

Comment:
maybe expose as the more standard `close()`?

Reply:
the name is taken from the issue description. I don't have any opinion about which name is better. Maybe maintainer @juanjux should decide?

Reply:
I agree, close() is better (same as files sockets, etc)